app/views/candidate_interface/shared/_need_help.html.erb

```diff
@@ -1,7 +1,6 @@
 <aside class="app-related" role="complementary">
   <h2 class="govuk-heading-s govuk-!-margin-bottom-2" id="help-title">Need help?</h2>
-
   <p class="govuk-body-s">
-    Call <%= govuk_link_to 'Get into Teaching', 'https://getintoteaching.education.gov.uk' %> on Freephone 0800 389 2500 or chat to an adviser using their <%= govuk_link_to 'online chat service', 'https://getintoteaching.education.gov.uk/lp/live-chat' %> between 8am and 8pm, Monday to Friday
+    Email us on <%= bat_contact_mail_to %> and we can give you personalised&nbsp;support.
```

Comment:
```suggestion
    Email us and we can give you personalised&nbsp;support:<br /><%= bat_contact_mail_to %> 
```
Prefer to avoid having a wrapping email address within text.